src/main/java/com/dungeoncrawler/Javiarenka/dungeonMapGenerator/webStuff/stylesTemplate.css

```diff
@@ -0,0 +1,34 @@
+html, body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.CORRIDOR {
+    background-color: #A9A9A9;
+}
+
+.WALL, .OBSTRUCTION{
+    background-color: rgb(70, 70, 70);
+}
+
+.ROOM {
+    background-color: #BDB76B;
+}
+
+.DOOR {
+    background-color: red;
+}
+
+.tile {
+    color: rgba(0,0,0,0);
+    font-size: 10px;
+    /* border: rgb(0,0,0) solid; */
+    /* border-width: 1px; */
```

Comment:
W razie czego border można skrócić do jednej linijki, tj. border: 1px solid rgba (0,0,0,0);

Tylko zaznaczę jeszcze, że wartość a w rgba odpowiada za kanał alpha, czyli przezroczystość. Jeśli jest 1, tzn że coś jest nieprzezroczyste, jeśli ma 0.5 to jest półprzezroczyste a jak ma 0 to jest absolutnie przezroczyste.

Reply:
Acha i jeszcze jedno na przyszłość może się przydać: wartość color odnosi się do koloru fonta. Jeśli chcemy, żeby div miał jakiś kolor, to używamy background-color. Tylko też div musi mieć jakąś wysokość bo bazowo równa się ona 0.